src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json

```diff
@@ -8,8 +8,8 @@
       "OpenCover": "4.6.166",
       "ReportGenerator": "2.3.1",
 
-      "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0023",
-      "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0023",
+      "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0025",
```

Comment:
I assume you will also update this in the test-runtime project in corefx as well? This coupling between the target files and the test-runtime\project.json seems like it is asking for some trouble. At a minimum I think it is worth allowing  folks to override your PackageVersion properties in the targets file. That way folks that have their own test-runtime have a choice of either overriding the versions used or updating their test-runtime project.json.